composeApp/src/commonMain/kotlin/org/ooni/engine/models/OONINetTest.kt

```diff
@@ -0,0 +1,10 @@
+package org.ooni.engine.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class OONINetTest(
```

Comment:
Their values match, but this is serializable version for the Engine API, and storing in the database. The `NetTest` is the pretty version to use in the app, that we can change freely.

Could you share how would you extend `NetTest`?

Reply:
If thats the case then we can leave it the way it is. Having the classes with almost no difference didn't sit right.

Reply:
Yeah, they're pretty close, I just created the other one because of the NetTestType, and to avoid having to write custom serializers for the Database and the Json.

Reply:
We can leave it as is.